### ISSUE TITLE
airmon-ng syntax error in SVN181

### ISSUE THREAD

**Issue by jbenden** [jbenden/aircrack-ng-migration#111]
**Reported by darkAudax on 16 Feb 2007 20:38 UTC**

Here is the error message from airmon-ng SVN181:

```
airmon-ng start wlan0 6

Interface       Chipset         Driver

wifi0           Atheros         madwifi-ng
/usr/local/sbin/airmon-ng: line 231: syntax error near unexpected token `('
/usr/local/sbin/airmon-ng: line 231: `         printf " (monitor mode enabled)"'
```

It was working in SVN161:

```
airmon-ng start wlan0 6

Interface       Chipset         Driver

wifi0           Atheros         madwifi-ng
ath0            Atheros         madwifi-ng VAP (parent: wifi0)
wlan0           RTL8187          (monitor mode enabled)
```

Migrated-From: https://trac.aircrack-ng.org/ticket/111

**Comment by jbenden** [comment]
**Modified by misterx on 16 Feb 2007 20:38 UTC**

**Comment by jbenden** [comment]
**Comment by misterx on 16 Feb 2007 20:38 UTC**

(In #191) Fixed airmon-ng syntax error (Closes: #111)

**Comment by jbenden** [comment]
**Modified by misterx on 16 Feb 2007 20:38 UTC**
